src/igvfd/tests/test_upgrade_rodent_donor.py

```diff
@@ -37,3 +37,9 @@ def test_rodent_donor_upgrade_6_7(upgrader, rodent_donor_v6_with_parents, parent
                              current_version='6', target_version='7')
     assert 'parents' not in value
     assert value['notes'] == 'This is a note.  parents: ' + parent_rodent_donor_1['@id']
+
+
+def test_rodent_donor_upgrade_7_8(upgrader, rodent_donor_v7):
+    value = upgrader.upgrade('rodent_donor', rodent_donor_v7, current_version='7', target_version='8')
+    assert value['schema_version'] == '8'
+    assert 'virtual' in value
```

Comment:
assert value of virtual

Reply:
Assert comparison is updated to compare the property to default property value of False.